### PR TITLE
preview 2.0 UI changes

### PR DESCRIPTION
**Pull request by pvorozhe**
Implementing ui changes from Preview UI 2.90 [Figma](https://www.figma.com/file/dBJQZDRtVJyWgo9TkBbnn6/Preview-2.0?node-id=1456%3A1573)

## Description:
1. Rearrange icons
2. Move functionality for the info button (from the Gatsby icon button)
3. Include "notification icon" and red dot for existing contifications
4. Minor code rearrangements

## Images

Screen Shot 2022-01-06 at 3 22 29 PM
Screen Shot 2022-01-06 at 3 22 19 PM
Screen Shot 2022-01-06 at 3 21 32 PM
Screen Shot 2022-01-06 at 3 21 25 PM
Screen Shot 2022-01-06 at 3 20 08 PM
Screen Shot 2022-01-06 at 3 44 05 PM
Screen Shot 2022-01-06 at 3 24 11 PM

### CODE REVIEW

File: packages/gatsby-plugin-gatsby-cloud/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-plugin-gatsby-cloud",
   "description": "A Gatsby plugin which optimizes working with Gatsby Cloud",
-  "version": "3.1.0-next.0",
```
Comment (TylerBarnes) [inline packages/gatsby-plugin-gatsby-cloud/package.json:4]: I think we'll want to revert changes to this file

**Review by TylerBarnes — COMMENTED** [review]
Nice :D will the html diffing logic when builds complete be part of this PR? If not, we should wait on this change https://github.com/gatsbyjs/gatsby/pull/34440/files#diff-5c04c0efa41cd68622ac03da3d8d7b6144d680bec06b6bc36f04b4cae1b3bb8cL49

**Comment by pvorozhe** [comment]
> Nice :D will the html diffing logic when builds complete be part of this PR? If not, we should wait on this change https://github.com/gatsbyjs/gatsby/pull/34440/files#diff-5c04c0efa41cd68622ac03da3d8d7b6144d680bec06b6bc36f04b4cae1b3bb8cL49

good call. I want to address it as a separate pr, im not 100% sure its a plugin-only change.

File: packages/gatsby-plugin-gatsby-cloud/src/components/buttons/IndicatorButton.js [hunk 2]
```diff
@@ -32,15 +32,27 @@ export default function IndicatorButton({
       >
         <div
           data-testid={`${testId}-button`}
+          onClick={
+            hoverable
+              ? onClick
+              : () => {
+                  setShowTooltip(!showTooltip)
+                }
```
Comment (DanielSLew) [inline packages/gatsby-plugin-gatsby-cloud/src/components/buttons/IndicatorButton.js:40]: Just a nit but not a huge fan of inlining functions like this, I think it makes it harder to read even if they're straightforward

File: packages/gatsby-plugin-gatsby-cloud/src/components/buttons/IndicatorButton.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react"
 import { IndicatorButtonTooltip } from "../tooltips"
-import { spinnerIcon } from "../icons"
+import { spinnerIcon, exitIcon } from "../icons"
+import { props } from "bluebird"
```
Comment (DanielSLew) [inline packages/gatsby-plugin-gatsby-cloud/src/components/buttons/IndicatorButton.js:4]: What's this for? we don't use it anywhere

File: packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js [hunk 4]
```diff
@@ -16,13 +30,48 @@ const getButtonProps = props => {
...
+          <BuildSuccessTooltipContent
+            isOnPrettyUrl={isOnPrettyUrl}
+            sitePrefix={sitePrefix}
+            buildId={buildId}
+            siteId={siteId}
+            orgId={orgId}
+          />
+        ),
+        active: true,
+        showInfo: true,
```
Comment (DanielSLew) [inline packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js:49]: Do we show an exit button everytime `showInfo` is `true`? If that's the case then we don't need both props. Do we need to introduce a new prop for the exit button or can we handle it another way? Even adding it directly onto the `BuildSuccessTooltipContent` could work.

File: packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js [hunk 5]
```diff
@@ -39,14 +88,12 @@ export default function InfoIndicatorButton(props) {
...
     })
   }
-
   return (
     <IndicatorButton
       testId="info"
-      iconSvg={infoIcon}
-      onMouseEnter={buttonProps?.active && trackHover}
+      iconSvg={buttonProps?.showInfo ? infoIconActive : infoIcon}
+      onClick={buttonProps?.active && trackHover}
```
Comment (DanielSLew) [inline packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js:95]: Does this make sense? We're tracking the hover when the button is being clicked now?

File: packages/gatsby-plugin-gatsby-cloud/src/components/buttons/LinkIndicatorButton.js [hunk 6]
```diff
@@ -61,6 +61,7 @@ export default function LinkIndicatorButton(props) {
     }, 2400)
 
     if (window) {
+      console.log(`copied`)
```
Comment (DanielSLew) [inline packages/gatsby-plugin-gatsby-cloud/src/components/buttons/LinkIndicatorButton.js:64]:
```suggestion
```

**Review by DanielSLew — APPROVED** [review]

**Pull request merged**